tests/lib/test_io.py

```diff
@@ -1,204 +1,160 @@
-import os
-from pathlib import Path
-import shutil
-import subprocess
+# pylint: disable=missing-class-docstring disable=missing-function-docstring disable=missing-module-docstring
```

Comment:
I think you can remove them since https://github.com/ChauffeurPrive/nestor-api/pull/33 got merged